BLSQ/iaso
# IA-3463 improve payment lot creation

Explain what problem this PR is resolving

Related JIRA tickets : IA-3463

## Self proofreading checklist


## Doc
In the code

## Changes

- Move all code related to payment lot creation to new task "create_payment_lot"
- Make sure payment lot is deleted if there's any error with the conversion of any potential payment into a payment
- Add "task" field to potential payment
- Add migration
- Add custom message when creating payment lot
- Add and update tests

## How to test

- Make sure to have at least 1 potential  payment, i.e.: have accepted org unit change requests
- Select one or several potential payments and create a payment lot

Then:
- Selected potential payments should have disappeared from table
- In task screen: a successful task should be visible
- In payment lots screen: the new payment lot should be visible


## Print screen / video

https://github.com/user-attachments/assets/6988e808-8bc5-417e-8ef7-9aa3028ddb4b
